public/app/plugins/datasource/prometheus/querybuilder/components/metrics-modal/ResultsTable.tsx

```diff
@@ -36,15 +37,7 @@ export function ResultsTable(props: ResultsTableProps) {
   function selectMetric(metric: MetricData) {
     if (metric.value) {
       onChange({ ...query, metric: metric.value });
-      reportInteraction('grafana_prom_metric_encycopedia_tracking', {
-        metric: metric.value,
-        hasMetadata: state.hasMetadata,
-        totalMetricCount: state.totalMetricCount,
-        fuzzySearchQuery: state.fuzzySearchQuery,
-        fullMetaSearch: state.fullMetaSearch,
-        selectedTypes: state.selectedTypes,
-        letterSearch: state.letterSearch,
-      });
+      tracking('grafana_prom_metric_encycopedia_tracking', state, metric.value);
```

Comment:
We should have a different label, but for now, I argue that we should keep this one until our testing is complete. This metric has been in use for testing so far so changing it would add a layer of complexity to our testing. I believe we can change it to the correct name once it is GA so that we can differentiate it from the early testing phase.

`grafana_prometheus_metrics_explorer_tracking` for example